cgeo-contacts/project/playstore/description/_source/app_description_long.txt

```diff
@@ -0,0 +1,12 @@
+This is a contacts plugin for c:geo, an open source geocaching app. It will only work with the c:geo app installed and allows direct access to your phonebook when clicking on a username in c:geo.
+
+Usage:
+When you click on a username in c:geo (e.g. in the cache logbook) the menu will contain an item to open the corresponding contact card in your phone contacts. This will allow you to directly call or message the user.
+
+Remark: 
+The username of the person needs to be stored in the name- or alias-field of your contacts in order for the plugin to match the geocaching username to the contact.
+
+Comments in PlayStore can not be answered! If you have problems installing or using c:geo please first have a look at our FAQ: http://faq.cgeo.org
```

Comment:
```suggestion
Comments in PlayStore can not be answered! If you have problems installing or using c:geo please first have a look at our FAQ: https://faq.cgeo.org.
```

Reply:
This was just an 1:1 import from Google Play. Once merged it can be updated via Crowdin.